docusaurus/docs/Android/02-client/06-guides/01-push-notifications/01-overview.mdx

```diff
@@ -0,0 +1,113 @@
+---
+slug: /client/guides/push-notifications/
+---
+
+# Overview
+
+Besides [Events](https://getstream.io/chat/docs/android/event_listening/?language=kotlin), push notifications are another way to stay up to date with changes in your chat in real time.
+
+:::info
+Members of a channel will receive a push notification if they don't have an open socket connection.
+:::
+
+To receive push notifications from Stream Chat, you'll need to:
+
+1. Configure your push notification provider on the [Stream Dashboard](https://dashboard.getstream.io/).
+2. Add the client-side integration for the chosen provider in your app.
+
+We support the following providers:
+
+- [Firebase](./02-firebase.mdx)
+- Huawei Push Kit (Coming soon, stay tuned!)
+- Mi Push (Coming soon, stay tuned!)
+
+We ship an individual artifact for each of these to make client-side integration with their service quick and simple. See their individual documentation pages linked above for details.
+
+## Customizing Push Notifications
```

Comment:
Common docs that apply to all push providers go here, under the intro of the main Push Notifications page.